asset/cloud-client/src/main/java/com/example/asset/ExportAssetsExample.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
Nit - here and elsewhere the exact line should be

`Copyright 2018 Google LLC`

Note no period at the end, which the pom.xml file has. The checkstyle copyright check gets very complainy about these, otherwise I wouldn't care.